exporter/collector/metrics.go

```diff
@@ -647,7 +648,7 @@ func (m *metricMapper) exponentialHistogramToTimeSeries(
 	_ pdata.ExponentialHistogram,
 	point pdata.ExponentialHistogramDataPoint,
 ) []*monitoringpb.TimeSeries {
-	if point.Flags().HasFlag(pdata.MetricDataPointFlagNoRecordedValue) {
+	if point.Flags().HasFlag(pmetric.MetricDataPointFlagNoRecordedValue) {
```

Comment:
just checking, this doesn't apply to exponential histograms?

Reply:
At least right now, exponential histograms' sum is [not optional](https://github.com/open-telemetry/opentelemetry-proto/blob/main/opentelemetry/proto/metrics/v1/metrics.proto#L541), whereas fixed bucket histograms' sum [is optional](https://github.com/open-telemetry/opentelemetry-proto/blob/main/opentelemetry/proto/metrics/v1/metrics.proto#L462).

Reply:
Ah thanks